components/contributors.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import glamorous, {Div} from 'glamorous'
+import {Anchor} from '../components/styled-links'
+
+const Container = glamorous.div({
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  '& > h3': {
+    margin: '20px',
+  },
+})
+
+const Img = glamorous.img({
+  height: 45,
+  width: 45,
+  borderRadius: '100%',
+  margin: '0 5px',
+  transition: 'transform .3s',
+  ':hover': {
```

Comment:
Just one quick question, is it possible to combine those 3 or do I have to do it this way
```
':focus': {
    transform: 'scale(1.2)',
  },
  ':hover': {
    transform: 'scale(1.2)',
  },
  ':active': {
    transform: 'scale(1.2)',
  },
```